examples/gin/README.md

```diff
@@ -2,10 +2,28 @@
 
 This is an example of basic Gin app with Airbrake middleware that reports route stats.
 
-## How to run API
+## How to run Example API
+
+Copy the example in your testing(eg. ginbrake) folder.
```

Comment:
What is a "testing (e.g. ginbrake) folder"? Does a user really need to copy it? or can they just run it from the example directory?